src/gearbox.cpp

```diff
@@ -69,6 +69,7 @@ Gearbox::Gearbox() {
         };
     }
     this->pressure_mgr = new PressureManager(&this->sensor_data, this->gearboxConfig.max_torque);
+    pressure_manager = this->pressure_mgr;
```

Comment:
What is the purpose of this?

Reply:
pressure_mgr is a global reference (Accessible to the whole TCU). I will (In a few commits), remove the reference of `this->pressure_mgr` in the Gearbox class, so everything can use the global pressure manager rather than the gearbox having to pass its pointer around to everyone that wants to use it